src/mscorlib/src/System/Collections/Concurrent/ConcurrentDictionary.cs

```diff
@@ -1987,14 +1973,7 @@ private ReadOnlyCollection<TValue> GetValues()
         [Conditional("DEBUG")]
         private void Assert(bool condition)
```

Comment:
I have decided against doing that -- I don't think that the code churn to desktop is worth it.